### ISSUE TITLE
Date params - examples in descriptions

### ISSUE THREAD

**Issue by kyyberi** [6aika/api-palaute#1]
When ever there is a date, an example of the format in the description would make it easier to adapt. Ref to standard could be added too.  Now I don't have a glue what is expected. 

```
      requested_datetime:
        type: string
        format: date-time
        description: Date and time when the service request was made.
```

**Comment by aapris** [comment]
Closed in 89c6d7cf406213947b2ebd4554b3264f0fa08733

**Comment by kyyberi** [comment]
It would have been nice to know if this was considered, implemented or something :)  Yes, I can see the link to commit, but still.
